components/Tabs/Tabs.js

```diff
@@ -1,62 +1,33 @@
+class Link{
+  constructor(element){
+    this.data;
+    this.itemElement;
+    this.select(element)
+  }
```

Comment:
Very good job on experimenting with different ways of doing the same thing. 
Your code here is shorter here then in your test.js file. That's good!